sources/zh.mangabz/res/source.json

```
@@ -0,0 +1,15 @@
+{
+	"info": {
+		"id": "zh.mangabz",
+		"name": "Māngabz",
+		"version": 5,
+		"urls": ["https://mangabz.com/"],

```

if there's just one url I believe you just use the `url` field, and make sure there's no trailing slash.
```suggestion
		"url": "https://mangabz.com",
```